scripts/src/java/org/oppia/android/scripts/TestFileCheck.kt

```diff
@@ -0,0 +1,84 @@
+package org.oppia.android.scripts
+
+/**
+ * Script for ensuring that every prod file has a
+ * corresponding Test file present in the repo.
+ */
+class TestFileCheck {
+  companion object {
+    @JvmStatic
+    fun main(vararg args: String) {
+      // path of the repo to be analyzed.
+      val repoPath = args[0] + "/"
+
+      // a list of all allowed directories in the repo to be analyzed.
+      // args[0] is the repoPath, the allowed directories are specified
+      // after it. Hence, we have to start from the 1st index.
+      val allowedDirectories = args.drop(1)
+
+      // a list of all kotlin files in the repo to be analyzed.
+      val searchFiles = RepoFile.collectSearchFiles(
+        repoPath,
+        allowedDirectories,
+        ".kt",
+        ExemptionsList.TEST_FILE_CHECK_EXEMPTIONS_LIST
+      )
+
+      // class object which is needed to acess the helper methods
+      val testFileCheck: TestFileCheck = TestFileCheck()
+
+      // a list of all the prod files present in the repo.
+      val prodFilesList: MutableList<String> = ArrayList()
```

Comment:
```suggestion
      val prodFilesList = mutableListOf<String>()
```

More idiomatic. Ditto elsewhere.

Reply:
Outdated.